backend/copr_backend/pulp.py

```diff
@@ -187,6 +187,19 @@ def delete_content(self, repository, artifacts):
         data = {"remove_content_units": artifacts}
         return requests.post(url, json=data, **self.request_params)
 
+    def copy_content(self, repository, units):
+        """
+        Copy a list of content units (RPM packages) into this repository
+        The "copy" isn't the right word because the actual data isn't touched
+        at all. Only a new database entry pointing to the data is created.
+        So regardless of the number of units, this operation should be fast.
+        https://pulpproject.org/pulp_rpm/restapi/#tag/Repositories:-Rpm/operation/repositories_rpm_rpm_modify
+        """
+        path = os.path.join(repository, "modify/")
```

Comment:
This `modify/` suffix deserves an in line comment :-)

Reply:
the endpoint (with the suffix) is mentioned in the documented link 

```
http://localhost:24817{rpm_rpm_repository_href}modify/
```